helpers/compile/plugins/fill-plugin/fillPlugin.ts

```diff
@@ -139,10 +139,11 @@ const defaultFillersConfig: Fillers = {
   path: { imports: path.join(__dirname, 'fillers', 'path.ts') },
   tty: { imports: path.join(__dirname, 'fillers', 'tty.ts') },
   util: { imports: path.join(__dirname, 'fillers', 'util.ts') },
+  crypto: { imports: path.join(__dirname, 'fillers', 'crypto.ts') },
+  'node:crypto': { imports: path.join(__dirname, 'fillers', 'crypto.ts') },
```

Comment:
I hope we can get rid of this in the future, and use `globalThis.crypto` only throughout our codebase

Reply:
I had to add this not because of our code but because of third-party dependencies (cuid and uuid generators)